pkg/config/protocol/instance.go

```diff
@@ -54,6 +54,8 @@ const (
 	Redis Instance = "Redis"
 	// MySQL declares that the port carries MySQL traffic.
 	MySQL Instance = "MySQL"
+	// HBONE declares that the port carries HBONE traffic.
+	HBONE Instance = "HBONE"
```

Comment:
Can we not add it here

Reply:
We would have to update all of our tests and echo codebase. Any concerns ? It cannot be used by real Service (which I plan to add a comment and test asserting this)

Reply:
My  concern is test code should not couple with main. Could be confused at a new protocol?

Reply:
I think "experimental" is more accurate than "test". This only exists in this file. Seems that an adequate "proceed at your own risk" banner should be sufficient, no?